source/common/router/router.cc

```diff
@@ -928,16 +928,25 @@ void Filter::UpstreamRequest::setupPerTryTimeout() {
 }
 
 void Filter::UpstreamRequest::onPerTryTimeout() {
-  ENVOY_STREAM_LOG(debug, "upstream per try timeout", *parent_.callbacks_);
-  parent_.cluster_->stats().upstream_rq_per_try_timeout_.inc();
-  if (upstream_host_) {
-    upstream_host_->stats().rq_timeout_.inc();
-  }
-  resetStream();
-  request_info_.setResponseFlag(RequestInfo::ResponseFlag::UpstreamRequestTimeout);
-  parent_.onUpstreamReset(
-      UpstreamResetType::PerTryTimeout,
-      absl::optional<Http::StreamResetReason>(Http::StreamResetReason::LocalReset));
+  // If we've sent anything downstream but per try timeout occurred
```

Comment:
nit: Can you reflow this comment to 100 cols. Also, rephrase:
"If we've sent anything downstream, ignore the per try timeout and let the response continue up to the global timeout."